src/Trixi.jl

```diff
@@ -66,7 +66,7 @@ include("auxiliary/auxiliary.jl")
 include("auxiliary/mpi.jl")
 include("auxiliary/p4est.jl")
 include("equations/equations.jl")
-include("mesh/mesh.jl")
+include("mesh/meshes.jl")
```

Comment:
In this case I think we should also rename `mesh/` -> `meshes/` to remain consistent, shouldn't we? Although `meshes` looks really really ugly, so I could also live with just `mesh` for both :grimacing: Consistent would be `meshes`, though :shrug: ...

Reply:
abf14cf2